### PR TITLE
FTBFS fix for loongarch batch #20

### PR DESCRIPTION
**Pull request by jiegec**
Topic Description
-----------------

- firejail: update to 0.9.72
- firebird-emu: update to 1.6
- figlet: migrate to ab4
- fflas-ffpack: update to 2.5.0
- givaro: update to 4.2.0

Package(s) Affected
-------------------

- figlet: 2.2.5-1
- fflas-ffpack: 2.5.0
- givaro: 4.2.0
- firebird-emu: 1.6
- firejail: 0.9.72

Security Update?
----------------

No

Build Order
-----------

```
#buildit givaro fflas-ffpack figlet firebird-emu firejail
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

**Experimental Architectures**

- [x] MIPS R6 64-bit (Little Endian) `mips64r6el`

### CODE REVIEW

**Comment by jiegec** [comment]
image
image
image
image
image

File: runtime-scientific/fflas-ffpack/autobuild/defines [hunk 1]
```diff
@@ -4,3 +4,6 @@ PKGDEP="givaro lapack"
 PKGDES="A library for dense linear algebra over word-size finite fields"
 
 AB_FLAGS_O3=1
+
+AUTOTOOLS_DEF+=(--with-blas-libs="-lcblas -llapack -lgomp")
```
Comment (MingcongBai) [inline runtime-scientific/fflas-ffpack/autobuild/defines]: Why not set this in `AUTOTOOLS_AFTER`?

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: runtime-scientific/fflas-ffpack/autobuild/defines (hunk 1 above)
Comment (jiegec) [reply runtime-scientific/fflas-ffpack/autobuild/defines]: AUTOTOOLS_AFTER cannot handle `"` properly.

Comment (jiegec) [reply runtime-scientific/fflas-ffpack/autobuild/defines]: `AUTOTOOLS_AFTER="--with-blas-libs=\"-lcblas -llapack -lgomp\""` does not work. `AUTOTOOLS_AFTER="--with-blas-libs=\\\"-lcblas -llapack -lgomp\\\""` neither.

Comment (jiegec) [reply runtime-scientific/fflas-ffpack/autobuild/defines]: Fixed.

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**